### DISCUSSION TITLE
Current balance only account

### DISCUSSION THREAD

**Discussion by Neurocranium** [firefly-iii/firefly-iii#6932] [Firefly III]
Hi folks,

would someone of you know of a way how to do a "balance only" account in firefly? So, an account which is getting synced from my bank but only shows the current balance instead of any transactions.

As a little background for this. My significant other and I have a shared account for our expenses but both also have a "pocket money" account. On one hand we would like the balance of our pocket money accounts in firefly to get a complete picture of our net worth but on the other hand, we don't want the transactions of our pocket money accounts to show up. Otherwise it is getting a lot harder to surprise the other with the occasional nice gift. :-)

Plus something like this could probably also be used to track an (ETF) depots balance to get it included in the net worth.

Best regards,
Sebastian

**Comment by JC5** [comment]
Getting synced would require you to script something yourself, but you could always create an account and play with the opening balance or virtual balance.
